integration-tests/test_client.py

```diff
@@ -82,21 +82,37 @@ def test_verify_logrotate_feature(insights_client):
     Ref : https://bugzilla.redhat.com/show_bug.cgi?id=1940267
 
     Test Steps:
-        1-Count the number of log file in beginning
-        2-Rotate logs by running logrotate command on CLI for insights-client
-        3-Verify that log is rotated and new log file is created
+        1 - Perform register and payload operation to ensure both
+            insights-client.log and insights-client-payload.log files have logs
+        2 - Rotate logs by running logrotate command on CLI for insights-client
+        3 - Verify insights-client.log and insights-client-payload.log size is 0B
+        4 - Verify rotated files are created in log dir by comparing number of log files
     """
 
     logrotate_conf_file_path = "/etc/logrotate.d/insights-client"
     logdir = "/var/log/insights-client/"
+    logfile_insights = "/var/log/insights-client/insights-client.log"
+    logfile_payload = "/var/log/insights-client/insights-client-payload.log"
```

Comment:
rather than duplicating "/var/log/insights-client", you can do something like:
```python
       logfile_insights = f"{logdir}/insights-client.log"
```